flambe/compile/component.py

```diff
@@ -368,7 +368,11 @@ def __init__(self, obj_id: str, value: Any = None):
 
     def __call__(self, stash: Dict[str, Any]) -> Any:
         if self.obj_value is not None:
-            if self.obj_value != stash[self.obj_id]:
+            diff_obj_stash = self.obj_value != stash[self.obj_id]
+            is_tensor = type(diff_obj_stash) == torch.Tensor
```

Comment:
I would recommend using `isinstance` rather than equality on the type in general unless you have a specific reason for not doing that here?

Reply:
It makes total sense! That way we also include subclasses. Changed!